src/ToricVarieties/LineBundles/constructors.jl

```diff
@@ -23,7 +23,7 @@ Construct the line bundle on the abstract normal toric variety `v` with class `c
 """
 function ToricLineBundle(v::AbstractNormalToricVariety, input_class::Vector{fmpz})
     class = picard_group(v)(input_class)
-    return ToricLineBundle(v, class)
+    return ToricLineBundle(v, class, Dict())
```

Comment:
I think it would be easier to add a default 2-argument constructor, see the "note" in https://nemocas.github.io/AbstractAlgebra.jl/dev/misc/#AbstractAlgebra.@attributes.

Reply:
Sorry for the confusion. This contains the changes of https://github.com/oscar-system/Oscar.jl/pull/941. Probably best to raised these points there.